data/decks/mole.json

```diff
@@ -0,0 +1,238 @@
+{
+	"id": "mole",
+	"season": 4,
+	"display_name": "Mole Knight",
+	"cardback": "cardback_mole.jpg",
+	"exceed_cost": 4,
+	"buddy_card": "burrow",
+	"buddy_display_name": "Burrow",
+	"buddy_exceeds": true,
+	"buddy_starting_offset": 0,
+	"character_action_default": [{
+		"gauge_cost": 0,
+		"force_cost": 0,
+		"effect": {
+			"effect_type": "place_buddy_at_range",
+			"buddy_name": "Burrow",
+			"range_min": 3,
+			"range_max": 3,
+			"and": {
+				"effect_type": "draw",
+				"amount": 1
+			}
+		}
+	}],
+	"character_action_exceeded": [{
+		"gauge_cost": 0,
+		"force_cost": 0,
+		"effect": {
+			"effect_type": "place_buddy_at_range",
+			"buddy_name": "Burrow",
+			"range_min": 2,
+			"range_max": 4,
+			"and": {
+				"effect_type": "draw",
+				"amount": 1
+			}
+		}
+	}],
+	"ability_effects": [
+		{
+			"timing": "set_strike",
+			"condition": "initiated_strike",
+			"effect_type": "choice",
+			"choice": [
+				{
+					"effect_type": "set_used_character_bonus",
+					"character_effect": true,
+					"linked_effect": {
+						"timing": "before",
+						"condition": "buddy_space_unoccupied",
+						"condition_detail": "Burrow",
+						"effect_type": "move_to_buddy",
+						"buddy_name": "Burrow"
+					}
+				},
+				{ "effect_type": "pass" }
+			]
+		},
+		{
+			"timing": "before",
+			"condition": "used_character_bonus",
+			"effect_type": "pass",
+			"silent_effect": true,
+			"and": {
+				"condition": "buddy_space_unoccupied",
+				"condition_detail": "Burrow",
+				"effect_type": "move_to_buddy",
+				"buddy_name": "Burrow"
+			}
+		}
+	],
+	"exceed_ability_effects": [
+		{
+			"timing": "set_strike",
+			"condition": "initiated_strike",
+			"effect_type": "choice",
+			"choice": [
+				{
+					"effect_type": "set_used_character_bonus",
+					"character_effect": true,
+					"linked_effect": {
+						"timing": "before",
+						"condition": "buddy_space_unoccupied",
+						"condition_detail": "Burrow",
+						"effect_type": "move_to_buddy",
```

Comment:
how does this interact with cannot move effects?
The text says "If you did, +2 power"
Did what? Tried to advance or actually advanced into the burrow?

Reply:
I checked, you only get the power if you successfully advance/retreat into the burrow (hence the on_buddy_space check)